pkgs/tools/misc/yutto/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, python3
+, ffmpeg
+, makeWrapper
+}:
+
+with python3.pkgs;
+
+buildPythonApplication rec {
+  pname = "yutto";
+  version = "2.0.0b13";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-XqfbvlaJoRPpwnaUA52KloOsEOIt3YM0cq36DZN6lR8=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    aiofiles
+    biliass
+    dicttoxml
+    colorama
+  ];
+
+
```

```suggestion
  pythonImportsCheck = [ module_name ];
```
See [the documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck) for more information

---

Thanks!